### PR TITLE
button update

### PR DESCRIPTION
**Pull request by AagmanSuri**

### CODE REVIEW

File: Components/Button/button.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+
+                <!-- Outline Button -->
+                <span class="sub-title">Outline Button</span>
+                <p class="p-content">In need of a button, but not the hefty background colors they bring? Replace
+                    '.btn-variant' with '.outline-variant'.</p>
+
+
+
+                <div class="button-container">
+                    <button class="btn outline">Defalt</button>
```
Comment (dhimanaishwarya) [inline Components/Button/button.html]: typo for default

File: Components/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+.outline-success:hover {
+  background-color: var(--success-color);
+  color: var(--white-color);
+  border: none;
+}
+
+.btn-icon {
+  position: absolute;
+  right: 64rem;
```
Comment (dhimanaishwarya) [inline Components/Button/button.css]: icon overlapping the text
image

File: Components/Button/button.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+.outline-secondary {
+  background-color: var(--white-color);
+  color: var(--secondary-color);
+  border: 1px solid var(--secondary-color);
+}
+
+.outline-secondary:hover {
+  background-color: var(--secondary-color);
+  color: var(--white-color);
```
Comment (SinchanaMS) [inline Components/Button/button.css:60]: The contrast between the background and the text is low, hindering the readability of secondary buttons. 

image

File: Components/Button/button.html (hunk 1 above)
Comment (j24m) [reply Components/Button/button.html]: +1

File: Components/Button/button.css (hunk 2 above)
Comment (j24m) [reply Components/Button/button.css]: +1

File: Components/Button/button.css (hunk 3 above)
Comment (j24m) [reply Components/Button/button.css:60]: As @SinchanaMS  said , you can either change the color of your secondary button or change the color of your background container. Rest is good:)

**Review by m26unkwn — APPROVED** [review]
Awsm code Quality
Loved It

File: Components/Button/button.html (hunk 1 above)
Comment (AagmanSuri) [reply Components/Button/button.html]: Typo resolved .Thanks for review

File: Components/Button/button.css (hunk 2 above)
Comment (AagmanSuri) [reply Components/Button/button.css]: Issue resolved .Thanks for review

File: Components/Button/button.css (hunk 3 above)
Comment (AagmanSuri) [reply Components/Button/button.css:60]: Issue resolved . Thanks for the review

**Comment by AagmanSuri** [comment]
Thank you [m26unkwn](https://github.com/m26unkwn)

**Pull request merged**